examples/example-nextjs/next.config.js

```diff
@@ -9,6 +9,45 @@
 
 /** @type {import('next').NextConfig} */
 
+const path = require('path');
+
 module.exports = {
   transpilePackages: ['@stylexjs/open-props'],
+  eslint: { ignoreDuringBuilds: true },
+  webpack: (config, { dev, isServer }) => {
+    // Process only files that require StyleX compilation using babel-loader
+    config.module.rules.push({
+      test: /\.(tsx|stylex\.ts|jsx|stylex\.js)$/,
```

Comment:
@km-tr 
This might be a bit too restrictive for general use as `.ts` and `.js` files may technically contain StyleX imports.

Might be better to just test for  `.js`, `.jsx`, `.ts`, `.tsx` in this example and let users fine-tune config according to their setup.

Reply:
Thank you. I've made it more generic.